_data/api.yml

```diff
@@ -58,7 +58,11 @@ paths:
   "/api/v1/namespaces":
     get:
       summary: Returns a list of namespaces
-      description: This will expose all accessible namespaces
+      description: |-
+        This will expose all accessible namespaces by the user via either team
+                                membership or visibility. Keep in mind that if the user is an admin, this
+                                will return all the global, personal and other namespaces created by all
+                                the users
```

Comment:
Is this alignment on purpose?

Reply:
Looks fishy, but on the rendered HTML it looks fine :sweat_smile: